ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1036,34 +1052,75 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
                                 style={{marginTop: '24px'}} largeHeader
                                 required={!this.isMode(WorkspaceEditMode.Duplicate)}>
           <FlexRow style={{alignItems: 'baseline'}}>
-            <TextInput type='text' style={styles.textInput} autoFocus placeholder='Workspace Name'
-              value = {name}
-              onChange={v => this.setState(fp.set(['workspace', 'name'], v))}/>
-            <TooltipTrigger
+            <FlexColumn>
+              <div style={styles.fieldHeader}>
```

Comment:
This header is new: https://projects.invisionapp.com/d/main#/console/20989573/442220263/preview